tests/TestKernelFactory.ts

```diff
@@ -76,6 +84,21 @@ export const testServices: Framework.ServiceList = new Map([
     ],
     [
         "infrastructure.test.inMemory.dbal",
-        { instance: ReadModel.InMemoryReadModelRepository },
+        { overwrite: true, instance: ReadModel.InMemoryReadModelRepository },
     ],
 ]);
+
+export async function TestKernelFactory(debug: boolean): Promise<Framework.Kernel> {
+    const TestModule = new Framework.ModuleContext({
+        services: testServices,
+        modules: [TransactionModule]
+    })
+
+    return await Framework.Kernel.createFromModuleContext(
```

Comment:
```suggestion
    return Framework.Kernel.createFromModuleContext(
```
Not sure if you need `await` here.

Do you do `return await` everywhere just so it's consistent?

Reply:
Yep but it's redundant, doesn't make sense really